src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/ReissuanceConsent/HealthCertifiedPersonReissuanceConsentViewController.swift

```diff
@@ -4,18 +4,28 @@
 
```

Comment:
Isn't there missing the handling of the `primaryButton's` and `secondaryButton's` actions? For the primary is also the loading indicator missing when tapping on it?